cluster/gce/gci/apiserver-configure-helper.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
For new files we add, there is no Kubernetes copyright. The same to other new files.